src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,103 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    private String Genre;
+    @Column(nullable = false)
+    private String autor;
+    @Column(nullable = false)
+    private String image;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String subtitle;
+    @Column(nullable = false)
+    private String publisher;
+    @Column(nullable = false)
+    private String year;
+    @Column(nullable = false)
+    private Integer pages;
+    @Column(nullable = false)
+    private String isbn;
+
+    Book() {
```

Comment:
In this line is missing the word "public" for identify that it is a constructor void.
Also missing the constructor with params

Reply:
I added a new constructor with params and the public access inn to the exist constructor